src/gdbstub_impl/ext/base.rs

```diff
@@ -131,16 +131,7 @@ impl<T: Target, C: Connection> GdbStubImpl<T, C> {
                 match xml {
                     Some(xml) => {
                         let xml = xml.trim().as_bytes();
-                        if cmd.offset < xml.len() {
-                            // still more data
-                            res.write_str("m")?;
-                            res.write_binary(
-                                &xml[cmd.offset..][..cmd.len.min(xml.len() - cmd.offset)],
-                            )?
-                        } else {
-                            // no more data
-                            res.write_str("l")?;
-                        }
```

Comment:
Can you use this two-branch implementation instead of the tri-branch implementation from memory_map? The tri-branch implementation is _technically_ more efficient at sending data over the wire, but it's not worth the additional binary bloat and code maintenance in `gdbstub`.